recipes/pandas_ml/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pandas_ml" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "a56ac33cae34b2db4a21da64cdb816ed30cb9e1d4fc97a1521bef1449532bd4d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas >=0.17.0
+    - enum34
```

Comment:
enum34 does not seem to be a `build` requirement ... just a run-time requirement